Gemfile

```diff
@@ -1,12 +1,16 @@
-# rubocop:disable Style/StringLiterals
+# rubocop:disable all
 source "https://rubygems.org"
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby "3.1.2"
```

Comment:
While you did a good job working on the project requirements I noticed you disabled rubocop. It is against microverse policy to disable linters because linters help us write clean code. So it would be nice if you enabled rubocop